tool.css

```diff
@@ -32,6 +32,10 @@ body {
 	width: 300px;
 }
 
+.FileColor {

```

Comment:
@MAZHARMIK - Let us use the style already present for orange text (search for `orange-text` in this file) and remove this style.